src/components/views/rooms/RoomHeader.js

```diff
@@ -32,6 +32,8 @@ import RoomTopic from "../elements/RoomTopic";
 import RoomName from "../elements/RoomName";
 import { PlaceCallType } from "../../../CallHandler";
 import { replaceableComponent } from "../../../utils/replaceableComponent";
+import exportConversationalHistory, { exportTypes, exportFormats } from '../../../utils/exportUtils/exportUtils';
```

Comment:
This import or some down the chain MUST be async otherwise you are now bundling the react-dom/server, and all new dependencies into the main bundle.